### ISSUE TITLE
Possibility for linux support

### ISSUE THREAD

**Issue by aurule** [BenjaminBini/barmaid#5]
Since it works in OSX and Windows, is it possible to get a version running in graphical linux desktops like Gnome and KDE? If possible, it would be a great feature.

**Comment by BenjaminBini** [comment]
Yes, definitely. I used Electron to make this app so it should work on Linux. I don't have any computer running Linux right now but I will install it on a VM and try to build and run it from there.

**Comment by aurule** [comment]
Excellent! 😄 I'll give it a try this weekend on my linux box.

**Comment by darkoverlordofdata** [comment]
Works beautifully in ElementaryOS.  After cloning barmaid to my GitHub folder (and tweaking the icon so its always white), I've added it in SystemSettings->Tweaks->Cerbere, so it's always running in my panel:

/home/user/.npm-global/bin/electron /home/user/GitHub/barmaid

![screenshot from 2016-11-18 19 11 11](https://cloud.githubusercontent.com/assets/1262194/20452627/2e6a1fb2-adc3-11e6-8624-c9940deaac76.png)

**Comment by aurule** [comment]
Awesome!

**Comment by BenjaminBini** [comment]
Great! Thank you very much!

(On another topic I added pug (jade) support a few months ago but did not have time to release a new version yet. If you manage to compile it by yourself, you can build it from the master branch.)

**Comment by BenjaminBini** [comment]
I built [the new version](https://github.com/BenjaminBini/barmaid/releases/tag/v1.1.0) for Linux, if someone wants to give it a try!
